data_architect_misc/data_transformer/configs/LATAM_Dominican_Republic/config.json

```diff
@@ -0,0 +1,126 @@
+      {
+        "__function_comment__": "The merged CSV file has unnamed columns. We need to drop them",
+        "function_name": "drop_unnamed_columns"
+      },
+      {
+        "__function_comment__": "By the time we run this function, there should be only 15 columns total remaining in the raw data frame.",
+        "__function_comment__": "this because the raw file contains the same structure as we come processing from the data tool.",
+        "function_name": "assert_number_of_columns_equals",
+        "function_args": [17]
+      },
+      {
+        "__function_comment__": "Add PROCESSED_DATE that holds the current date value.",
+        "function_name": "add_PROCESSED_DATE_column_with_current_date"
+      },
+      {
+        "__function_comment__": "Create HARMONIZED_YEAR column from existing date column name.",
+        "function_name": "add_HARMONIZED_YEAR_column_by_renaming_existing_column",
+        "function_args": ["Year"]
+      },
+      {
+        "__function_comment__": "Create HARMONIZED_MONTH column from existing date column name.",
+        "function_name": "add_HARMONIZED_MONTH_column_using_existing_month_column_with_only_full_month_names",
+        "function_args": ["Month"]
+      },
+      {
+        "__function_comment__": "Add DATE column based on the values in ***YEAR and MONTH columns created in previous steps***.",
+        "function_name": "add_HARMONIZED_DATE_column_using_existing_YEAR_and_MONTH_columns_with_integer_values"
+      },
+      {
+        "__function_comment__": "Add HARMONIZED_REGION as a new column with 'Asia Pacific' as value.",
+        "function_name": "add_HARMONIZED_REGION_column",
+        "function_args": ["Latin America"]
+      },
+      {
+        "__function_comment__": "Create HARMONIZED_COUNTRY column (with standardized country names in comp_harm_constants.py) from raw country name column.",
+        "function_name": "add_HARMONIZED_COUNTRY_column_using_fixed_str_value",
+        "function_args": ["Dominican Republic"]
+      },
+      {
+        "__function_comment__": "Create HARMONIZED_ADVERTISER column (with standardized advertiser names in comp_harm_constants.py) from existing raw advertiser name column.",
+        "function_name": "add_HARMONIZED_ADVERTISER_column_using_existing_advertiser_column",
+        "function_args": ["Producer"]
+      },
+      {
+        "__function_comment__": "Create HARMONIZED_MEDIA_TYPE column (with standardized Media Type names in comp_harm_constants.py) from raw media type column.",
+        "function_name": "add_HARMONIZED_MEDIA_TYPE_column_using_existing_media_type_column"
+      },
+      {
+        "__function_comment__": "Add CURRENCY as a new column with 'ZAR' as value.",
+        "function_name": "add_HARMONIZED_CURRENCY_column",
+        "function_args": ["DOP"]
+      },
+      {
+        "__function_comment__": "In HK data file, the rows under '$' column (spend values) sometimes have commas with them. We should remove them.",
```

Comment:
Is this HK or Dominican Republic? Please update the comments or simply delete them.